src/main/scala/io/iohk/ethereum/blockchain/sync/regular/ChainManagerService.scala

```diff
@@ -0,0 +1,28 @@
+package io.iohk.ethereum.blockchain.sync.regular
+
+import cats.data.NonEmptyList
+import io.iohk.ethereum.blockchain.sync.regular.BlockImporter.BlockImportType
+import io.iohk.ethereum.domain.Block
+import io.iohk.ethereum.network.Peer
+import monix.eval.Task
+
+//not used atm, a part of the future ExecutionSync
+object ChainManagerService {
```

Comment:
Is the idea to merge this PR as is, and have this modified in ETCM-1005 (Design new sync layer and chains manager) or later?

Reply:
yes, ETCM-1005 is an independent effort